plugin/trino-faker/pom.xml

```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.trino</groupId>
+        <artifactId>trino-root</artifactId>
+        <version>461-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>trino-faker</artifactId>
+    <packaging>trino-plugin</packaging>
+    <description>Trino - Faker connector</description>
+
+    <dependencies>
+
```

Comment:
`Missing dependency on io.airlift:junit-extensions in /home/runner/work/trino/trino/plugin/trino-faker/pom.xml`